### ISSUE TITLE
import dask_ml.joblib raises error

### ISSUE THREAD

**Issue by kwunlyou** [dask/dask-ml#591]
running `import dask_ml.joblib` will raise an error as

```python
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-63a6aa9c7464> in <module>
----> 1 import dask_ml.joblib  # registers joblib plugin

/usr/local/lib/python3.6/dist-packages/dask_ml/joblib.py in <module>
----> 1 import distributed.joblib  # noqa

ModuleNotFoundError: No module named 'distributed.joblib'
```

```
dask-ml              1.1.1
distributed          2.9.0
```

**Comment by TomAugspurger** [comment]
That's been removed. You just need to create a Client and use `joblib.parallel_backend` now

```python
from dask.distributed import Client
import joblib

client = Client(processes=False)             # create local cluster
# client = Client("scheduler-address:8786")  # or connect to remote cluster

with joblib.parallel_backend('dask'):
    # Your scikit-learn code
```

https://ml.dask.org/joblib.html

Are there any remaining references to `dask_ml.joblib`?

I'm a little confused by the error though. https://github.com/dask/dask-ml/pull/545 is what removed it, so there shouldn't even be a `dask_ml.joblib` file.

**Comment by TomAugspurger** [comment]
Intersting, there's a problem with the wheel. I must have had that file around locally when I built the wheel.

```
$ pip install dask-ml
Installing collected packages: dask-ml
Successfully installed dask-ml-1.1.1
$ python
>>> import dask_ml.joblib
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/private/tmp/ttt/lib/python3.7/site-packages/dask_ml/joblib.py", line 1, in <module>
    import distributed.joblib  # noqa
ModuleNotFoundError: No module named 'distributed.joblib'
```

If I install from source, things work as expected,

```
$ pip install --no-binary=dask-ml dask-ml
$ python
>>> import dask_ml.joblib
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'dask_ml.joblib'
```

I'll try to make a release with the fixed wheel this week.

**Comment by kwunlyou** [comment]
>Are there any remaining references to dask_ml.joblib?

I saw this use from your youtube video and read this old [doc](https://dask-ml.readthedocs.io/en/latest/joblib.html). Sorry for my bad. Didn't notice it has been moved.

**Comment by TomAugspurger** [comment]
No worries. Updating the video is hard, and I haven't figured out how to
redirect the old docs :)

I'll leave this issue open as a reminder to verify that dask_ml.joblib is
gone from the binaries in the next release.

**Comment by TomAugspurger** [comment]
This is fixed in dask-ml 1.2.0
